### ISSUE TITLE
JVM implementation

### ISSUE THREAD

**Issue by dgomesbr** [w3c/web-of-things-framework#5]
I saw the docs and the current implementations, is there any plan to have a JVM implementation of the framework?

I would suggest:
* Scala
* Finagle
* Zipkin
* Swagger
* Cassandra 
* Spark

It's ok to do a PR as a sub project?

**Comment by mpetyx** [comment]
I could suggest and contribute on:

* Swagger
* Spark

and further suggest

* Python
* Django 

Contribution that I could definitely start the implementations myself.
I had already mentioned Swagger https://github.com/w3c/web-of-things-framework/issues/3 earlier today and I think it is a top priority.

**Comment by dgomesbr** [comment]
Hey @mpetyx I've seen your comments on #3 and didn't knew Hydra. Since my expertise is into low end, JVM languages and distributed system I've imagined to create the same things as described in group and readme.

Since we're talking of lots of data, spark toolset looks promising (maybe storm or others may come together) integrated with a Kafka pipeline.

**Comment by draggett** [comment]
It would be great to have web of things servers implemented for other platforms and supporting different scripting languages, but note that we will need a test harness and test suite to demonstrate interoperability. So we should think about what that entails.

**Comment by pablochacin** [comment]
I would be interested in cotributing with Java 8 (functional).

**Comment by mpetyx** [comment]
Having a Swagger interface could make it much much easier to create test suites. Another useful approach since we are talking about REST services would be to have [POSTMAN](https://chrome.google.com/webstore/detail/postman-rest-client/fdmmgilgnpjigdojojpjoooidkmcomcm?hl=en) collections.
I could contribute to both if you'd like and in my opinion we could start developing scenarios on the framework that would be showcased with POSTMAN and through Swagger.
Maybe start working on this at the Wiki?

**Comment by dgomesbr** [comment]
Guys,

    Really loved the idea, I'll write both Java (7< and 8) and Scala impl over the weekend and post a few drafts in separated issues so we can evaluate both of then.

**Comment by dgomesbr** [comment]
@draggett yeah, I guess since it's main goal is adoptability we really should have a strong and consistent test suite over the interoperability.
